### ISSUE TITLE
narratives and dayplans are now connected via unit.uid and not any longer via unit.id

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#486]
and because of that legacy data must be changed: related dayplans and related narratives need to be rewired from id to uid after the migration has added the fields dayplan.uid_unit and narrative.uid_unit

**Comment by urapadmin** [comment]
### @test

- [ ] with version V8.9.7 you should be able to add dayplans and work with them like you are used to
- [ ] with version V8.9.7 you should be able to add unit narrative and work with them like you are used to
- [ ] existing dayplans and narratives should be there and behave normally

**Comment by lbestock** [comment]
A narrative hiccup. I created some with no problem. (Alice, URAP, 8.9.9.) I went to delete my most recent narrative. It said the image would remain, which is wrong since it is not an image. I said "yup" (without taking a screen shot of the error message, argh). I then got another error message: 
![image](https://user-images.githubusercontent.com/49291909/78829022-9ee2ec00-79b3-11ea-8c39-316796b9ad99.png)

I hit OK. And now the narrative is gone. But crucially I am also not able to create a new narrative in that unit. Next step, trying to delete the next narrative so I can recreate the first error message. That worked:

![image](https://user-images.githubusercontent.com/49291909/78829539-832c1580-79b4-11ea-8b77-97270e035549.png)

Next I tried if I could create a narrative in another unit or if that is kaputt across the board. Yes, I can create a new narrative in a different unit. 

Then I went back to FA to see if, having left, I could come back and make a narrative. And. There are four blank narratives atop the list. They're normal narrative blanks into which I can put text, assign a person and time. Are they because I hit the + button four times (I'm not sure how many times I hit it) trying to make new narratives a minute ago? And now I can make new ones, too, and it's normal.
